types/d3-array/index.d.ts

```diff
@@ -165,6 +165,24 @@ export function median<T>(
     accessor: (element: T, i: number, array: Iterable<T>) => number | undefined | null
 ): number | undefined;
 
+/**
+ * Returns the cumulative sum of the given iterable of numbers, as a Float64Array of the same length.
+ * If the iterable contains no numbers, returns zeros.
+ * An optional accessor function may be specified, which is equivalent to calling Array.from before computing the cumulative sum.
+ * This method ignores undefined and NaN values; this is useful for ignoring missing data.
+ */
+export function cumsum<T extends Numeric>(iterable: Iterable<T | undefined | null>): Float64Array;
+/**
+ * Returns the cumulative sum of the given iterable of numbers, as a Float64Array of the same length.
+ * If the iterable contains no numbers, returns zeros.
+ * An optional accessor function may be specified, which is equivalent to calling Array.from before computing the cumulative sum.
+ * This method ignores undefined and NaN values; this is useful for ignoring missing data.
+ */
+export function cumsum<T>(
+    iterable: Iterable<T>,
+    accessor: (element: T, i: number, array: Iterable<T>) => number | undefined | null
```

Comment:
Shouldn't `T`  be `T|undefined|null`?

Reply:
I didn't think about it too much since I just used followed the pattern of the already existing `sum`, `mean`, `median`, `quantile`, etc. However I think it's acceptable to just be `T`, since `T` can include `| undefined | null` if desired and it's really the output value that we care about/know about. We don't know anything about `T`.

Reply:
Ok, fine for me :-)